# Controller settings
float32 p_in
float32 i_in
float32 d_in
float32 bias
float32 i_accumulator_min
float32 i_accumulator_max
float32 output_min
float32 output_max
float32 deadband_min
float32 deadband_max

# Encoder settings
float32 target_update_rate
float32 max_abs_acceleration
float32 max_abs_velocity
bool second_order_predictor

# Motor settings
float32 smoothener_alpha